remoulade/cancel/middleware.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, *, backend=None):
         self.backend = backend
 
     def before_process_message(self, broker, message):
-        group_id = message.options.get("group_info", {}).get("group_id")
+        group_id = self.get_option_value("group_info", broker, message, {}).get("group_id")
```

Comment:
when using an input that is not a variable, I like to add the input name to have a more readable code, as here we don't know what `{}` is for. You can enforce this behavior by defining your method like this (what I was telling you about forcing the use of kwargs in a function):
```python
def get_option_value(self, option_name, broker, message, *, default=None):
```
I also prefer `default` to `default_value` as it is shorter and the same as a lot of python functions, like dict.get